include/rocksdb/compaction_filter.h

```diff
@@ -173,6 +173,13 @@ class CompactionFilter {
     return Decision::kKeep;
   }
 
+  // Internal (BlobDB) use only. Do not override in application code.
+  virtual bool PrepareBlobOutput(const Slice& /* key */,
+                                 const Slice& /* existing_value */,
+                                 std::string* /* new_value */) const {
+    return false;
+  }
```

Comment:
Can you explain why this cannot be done in FilterV2() too?

Reply:
Unless I'm mistaken `FilterV2` would get called for all blobs encountered during compaction, not just the surviving (still valid) ones.